### ISSUE TITLE
learn about the trajectory(s)" aka "ask for advice" page

### ISSUE THREAD

**Issue by 7yl4r** [7yl4r/the-oregon-trajectory#40]
learn about the trajectory(s)" aka "ask for advice" page where you are choosing 1mo, 2mo, 3mo

for now we ought to just focus on a draft of the content, later on we can convert the content to an html page

**Comment by JesusVidal** [comment]
Guys you are thinking of making differents rockets, I have seen the chemical one, Should I change it colour maybe to do it, more chemical?

**Comment by 7yl4r** [comment]
@JesusVidal, sounds good [here's a new issue](https://github.com/7yl4r/the-oregon-trajectory/issues/43).

**Comment by MrNubbin** [comment]
[Slides 3-5](https://docs.google.com/presentation/d/1trmPsQiFNQtOHnaCyKzaeCg-4O-M7eOH3YFqmbBeQ4A/edit#slide=id.g90d70e548_0_143) for the description of the game. Slide 13 for the advice on the trajectory.

**Comment by MrNubbin** [comment]
See #56 and update the storyboard accordingly

**Comment by MrNubbin** [comment]
The year is 2050 and you are tasked with a mission called Oregon, named after the Oregon Trail of the 1800s, to go to Jupiter’s moon Europa. NASA scientists discovered that Europa has signs of water and organic molecules such as methane. They believe life is under the surface of Europa and you’re going to find it!

Since the discovery of the potential for life on Europa, NASA has been ramping up it’s capabilities to get humans out of Earth’s orbit. This all started in the mid 2020’s with the development of the Space Launch System and with NASA’s Asteroid Redirect Mission to identify, capture and redirect a near-Earth asteroid to a stable orbit around the moon.

The Asteroid Redirect Mission (ARM) helped develop new technologies and hardware to reach these near-Earth asteroids. These technologies also made their way to the private space industry where companies were interested in mining these asteroids for metals that are rare on Earth. This was the start of a new age where astronauts were regularly traveling out of Earth’s orbit.

Everything has been setup for your mission to Europa. You will fly-by the Moon, Mars, and the asteroid Ceres along your way. There are manned stations on the Moon, Mars, and Ceres to buy new supplies. After that you will find a base with all the supplies you need waiting on the surface of Europa. You will also find old satellites that could hold additional fuel for you to salvage. Don’t forget to mine some asteroids along the way. Good luck!

**Comment by MrNubbin** [comment]
That is also [Slides 3-6](https://docs.google.com/presentation/d/1trmPsQiFNQtOHnaCyKzaeCg-4O-M7eOH3YFqmbBeQ4A/edit#slide=id.g90d70e548_0_143) of the storyboard. (also linked in the wiki). Closing.
